automates/text_reading/src/main/resources/application.conf

```diff
@@ -60,7 +60,8 @@ CommentEngine {
 
 apps {
 
-  projectDir = "/home/marco/repos/automates"
+projectDir = "/home/alexeeva/Repos/automates"
```

Comment:
@maxaalexeeva It would be great if we set this variable to a non-specific user default that would require all users to override the variable when starting a TextReading web app session. Maybe something like `projectDir = "/local/path/to/automates"`

Can we do that? Do you remember the command that allows a user to override an environment variable when running `sbt`?